ocaml/xapi/xapi_globs.ml

```diff
@@ -919,8 +919,8 @@ let xenopsd_queues = ref ([
 
 let default_xenopsd = ref "org.xen.xapi.xenops.xenlight"
 
-let ciphersuites_good_outbound = ref None
-let ciphersuites_legacy_outbound = ref ""
+let ciphersuites_good_outbound = ref "!EXPORT:RSA+AES128-SHA256"
```

Comment:
Hi @lindig 
Thanks for your comments.
I added change request into the commits.